blueprints/admin/routes.py

```
@@ -0,0 +1,253 @@
+                tags=tags,
+            )
+
+            db.session.add(new_article)
+            db.session.commit()
+            flash("Article Created Successfully!", "success")
+            return redirect(url_for("admin.admin_dashboard"))
+
+        except Exception:
+            db.session.rollback()
+            flash(
+                f"Something went terribly wrong while trying to add your article",
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)